### PR TITLE
Moving to new webview

### PR DESCRIPTION
**Pull request by antonmolodykh**
Refs #527

### CODE REVIEW

File: Dockerfile.viewer [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+RUN apt-get update && \
+    apt-get -y install \
+        build-essential \
+        curl \
+        git-core \
+        gstreamer0.10-plugins-good \
+        libdbus-glib-1-dev \
+        libffi-dev \
+        libqt5webkit5 \
```
Comment (vpetersson) [inline Dockerfile.viewer]: Do we need this given that we have compiled the binary in the other repo?

File: Dockerfile.viewer [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        python-netifaces \
+        sqlite3 \
+        x11-utils \
+        x11-xserver-utils \
+        xserver-xorg && \
+    apt-get clean
+
+# Install Python requirements
+ADD requirements.viewer.txt /tmp/requirements.txt
+RUN curl -s https://bootstrap.pypa.io/get-pip.py | python && \
```
Comment (vpetersson) [inline Dockerfile.viewer]: Do we really need this on Stretch? I would assume it is running a fairly recent version. it was a workaround on earlier version of Raspbian.

File: Dockerfile.websocket_server [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    apt-get -y install \
+        build-essential \
+        curl \
+        libffi-dev \
+        python-dev && \
+    apt-get clean
+
+# Install Python requirements
+ADD requirements.websocket_server.txt /tmp/requirements.txt
+RUN curl -s https://bootstrap.pypa.io/get-pip.py | python && \
```
Comment (vpetersson) [inline Dockerfile.websocket_server]: Same.

File: Dockerfile.viewer [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+RUN tar -xzf /tmp/screenly-webview-v0.1.tar.gz -C /tmp
+
+RUN cp -r /tmp/res /usr/local/share/ScreenlyWebview/res
+RUN cp /tmp/ScreenlyWebview /usr/local/bin/ScreenlyWebview
+
+# Fix white flickering in omxplayer
+ENV NOREFRESH 1
+
+# Copy X11 configs
+COPY ansible/roles/system/files/10-serverflags.conf /usr/share/X11/xorg.conf.d/10-serverflags.conf
```
Comment (vpetersson) [inline Dockerfile.viewer]: Let's move these file out of the Ansible folder to reduce confusion later.

File: ansible/roles/screenly/tasks/main.yml [hunk 5]
```diff
@@ -78,12 +63,64 @@
...
     group: root
 
-- name: Copy screenly systemd units
-  copy:
-    src: "{{ item }}"
-    dest: "/etc/systemd/system/{{ item }}"
+- name: Check if deprecated systemd services exists
+  stat:
+    path: /etc/systemd/system/X.service
+  register: X_service
```
Comment (vpetersson) [inline ansible/roles/screenly/tasks/main.yml]: X_service -> x_service

(to keep things consistent)

File: Dockerfile.viewer (hunk 1 above)
Comment (antonmolodykh) [reply Dockerfile.viewer]: yes, the binary file has static links to the files from these libraries

File: Dockerfile.viewer (hunk 2 above)
Comment (antonmolodykh) [reply Dockerfile.viewer]: yes, it is needed on Stretch. pip is not installed by default

Comment (vpetersson) [reply Dockerfile.viewer]: Instal the package `python-pip` install.

**Pull request closed without merging**